### PR TITLE
Bug fix for #60224 (nxos_vrf errors with reserved words)

### LINKED ISSUE #60224: nxos_vrf errors with reserved words
##### SUMMARY

Function `get_value` in `nxos_vrf` module https://github.com/ansible/ansible/blob/203caf2570e236e1569f7ebac738b41f51f1b02c/lib/ansible/modules/network/nxos/nxos_vrf.py#L381 doesn't handle using reserved words (`vni`, `rd`, `description`) as part of static route description or VRF description itself.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

lib/ansible/modules/network/nxos/nxos_vrf.py

##### ANSIBLE VERSION

```paste below
ansible 2.8.1
  config file = None
  configured module search path = [u'/home/ansible-network/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15 (default, Aug 16 2018, 14:17:09) [GCC 6.4.0]
```

##### CONFIGURATION

```paste below

```

##### OS / ENVIRONMENT

Cisco Nexus Switches [NXOS: version 7.0(3)I7(4)]

##### STEPS TO REPRODUCE

Create an VRF like:

```yaml
- name: "Ensure VRF MIDDLE route"
  connection: network_cli
  nxos_static_route:
    name: MIDDLE
    description: 'my_prd_environment_vrf'
    state: present
```

The same happen if the string `rd` or `vni` not followed by a white space is present in some VRF static route.

##### EXPECTED RESULTS

The execution should not break with a traceback

##### ACTUAL RESULTS

First execution will work and create needed resources, but the following executions will fail with exception:

```
Traceback (most recent call last):
  File \"/root/.ansible/tmp/ansible-local-64KxgDiV/ansible-tmp-1565195249.03-248614178720100/AnsiballZ_nxos_vrf.py\", line 114, in <module>
    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-local-64KxgDiV/ansible-tmp-1565195249.03-248614178720100/AnsiballZ_nxos_vrf.py\", line 106, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File \"/root/.ansible/tmp/ansible-local-64KxgDiV/ansible-tmp-1565195249.03-248614178720100/AnsiballZ_nxos_vrf.py\", line 49, in invoke_module
    imp.load_module('__main__', mod, module, MOD_DESC)
  File \"/tmp/ansible_nxos_vrf_payload_vhEzCs/__main__.py\", line 516, in <module>
  File \"/tmp/ansible_nxos_vrf_payload_vhEzCs/__main__.py\", line 501, in main
  File \"/tmp/ansible_nxos_vrf_payload_vhEzCs/__main__.py\", line 412, in map_config_to_obj
  File \"/tmp/ansible_nxos_vrf_payload_vhEzCs/__main__.py\", line 385, in get_value
AttributeError: 'NoneType' object has no attribute 'group'
```

### PR DESCRIPTION
**Pull request by adharshsrivatsr**
##### SUMMARY
Added fix to handle reserved errors in description in nxos_vrf module.

Fixes #60224

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/modules/network/nxos/nxos_vrf.py

### CODE REVIEW

File: lib/ansible/modules/network/nxos/nxos_vrf.py [hunk 1]
```diff
@@ -379,10 +379,12 @@ def map_params_to_obj(module):
 
 
 def get_value(arg, config, module):
```
Comment (Qalthos) [inline lib/ansible/modules/network/nxos/nxos_vrf.py:381]: What is the point of `arg` here, and how does it differ from `arg` below?

Comment (adharshsrivatsr) [reply lib/ansible/modules/network/nxos/nxos_vrf.py:381]: `arg` takes the value of the parameters (extra_params) in a loop in the `map_config_to_obj` method which calls the `get_value` method. Here, we check if arg is in the config as a key. Then we take its corresponding value.

Comment (Qalthos) [reply lib/ansible/modules/network/nxos/nxos_vrf.py:381]: Oh my goodness, I misread the `if` below as `for`. Repeatedly.

Ignore me.

**Review by ganeshrn — CHANGES_REQUESTED** [review]
Can you please add either integration or unit test for this scenario

**Pull request closed without merging**